dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
I have in mind, that I run in trouble when using an unnamed group in a unit test once. If I remember correctly, somewhere in our AuthorizeService or GroupService we still use group names instead of UUIDs. I think it would be appropriate to either through an error here. Maybe we should create another jira iisue about group without names?